packages/react-docs-example/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "@0xproject/react-docs-example",
+    "version": "0.0.1",
+    "private": true,
+    "description":
+        "An example app using react-docs",
+    "scripts": {
+        "lint": "tslint --project . 'ts/**/*.ts' 'ts/**/*.tsx'",
+        "build": "tsc",
+        "build:example": "NODE_ENV=production webpack",
+        "build:watch": "tsc -w",
+        "clean": "shx rm -rf lib; shx rm -f public/bundle*",
+        "dev": "webpack-dev-server --open",
+        "deploy_example":
+            "npm run build:example; aws s3 sync ./public/. s3://react-docs-example --profile 0xproject --region us-east-1 --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers"
```

Comment:
If it's intended to be forked by other people - I'll remove 0x-specific configs like region of profile name, but I also see it being easier for us and they'll probably be smart enough to figure it out.

Reply:
Yeah, think we'll keep it for now. Mentioned they'd need to update it in the README.